plottr/data/datadict.py

```diff
@@ -732,7 +732,12 @@ def add_data(self, **kw: Sequence):
         """
         dd = self.structure(same_type=True)
         for k, v in kw.items():
-            dd[k]['values'] = v
+            if isinstance(v, list):
```

Comment:
this change is solving the same issue as https://github.com/data-plottr/plottr/pull/68 , right?

Reply:
no, that's a different issue that #68 fixed -- but that issue is solved anyway by using get_parameter_data.